docs/source/quickstart/galaxy_simple.rst

```diff
@@ -0,0 +1,122 @@
+Quickstart: Galaxy with button-based tools
```

Comment:
simplified tools? or graphical interface? :D

Reply:
I kind of liked the button-based tools here. It is quickstart, so users come in without prior knowledge of the lingo. Galaxy in itself is a graphical interface so it could confuse people that YAML-based galaxy is not graphical. "button-based or YAML"  is more informative than "simplified or YAML", as it could also imply like a simplified YAML or something.